addons/medical_treatment/XEH_postInit.sqf

```diff
@@ -80,15 +101,21 @@ if (["ace_trenches"] call EFUNC(common,isModLoaded)) then {
             [1.05, 0.02, 0.3] //position in centre of cross
         ] call EFUNC(interact_menu,createAction);
 
-        ["Land_Grave_dirt_F", 0, [], _checkHeadstoneAction] call EFUNC(interact_menu,addActionToClass);
+        [missionNameSpace getVariable [QGVAR(graveClassname), "Land_Grave_dirt_F"], 0, [], _checkHeadstoneAction] call EFUNC(interact_menu,addActionToClass);
```

Comment:
I'm not sure about this
the action has a specific offset for that model and a "t" icon that we might not want to show on generic grave models

Reply:
Yeah the offset and icon will have to be updated when the new grave model @Marstruc is adding is merged.

Reply:
@PabstMirror a headstone icon like this suit you?
![5dcec0595d1b770f62bb2095c2a6e266](https://github.com/acemod/ACE3/assets/15274778/718012d2-8f29-4a52-b1b6-b147c0cb5f9a)